rule/utils.go

```diff
@@ -140,3 +140,81 @@ func isCallToExitFunction(pkgName, functionName string) bool {
 func newInternalFailureError(e error) []lint.Failure {
 	return []lint.Failure{lint.NewInternalFailure(e.Error())}
 }
+
+// isUpper checks if rune is digit
+func isDigit(r rune) bool {
+	return r >= '0' && r <= '9'
```

Comment:
Technically in go we can name the symbols in unicode. Should we use unicode.IsDigit instead? If we deliberately don't want to use the unicode package, I'd then put a comment on the reasoning.

Reply:
Please read resolved comments

Everything was almost about this remark

Such ad 
https://github.com/mgechev/revive/pull/1322#discussion_r2057855880

So yes, adding a comment and a unit test about a "unicode digit"

Reply:
@denisvmedia the comments were added by @mfederowicz